### PR TITLE
ROX-14074: Enable active vuln test for postgres

### PR DESCRIPTION
**Pull request by c-du**
## Description

Change active vuln test to use new image vuln interface.

## Checklist
- [ ] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

Run active vuln test.

### CODE REVIEW

File: tests/active_vuln_test.go [hunk 1]
```diff
@@ -114,119 +108,98 @@ func TestActiveVulnerability_SetImage(t *testing.T) {
...
+func getImageComponents(t *testing.T, query string, scopeQuery string) []ActiveComponent {
 	var resp struct {
-		Image ComponentsAndVulnsWithActiveState `json:"image"`
+		Components []ActiveComponent `json:"imageComponents"`
 	}
 	makeGraphQLRequest(t, `
-		query getImageCVE($imageID: ID!, $scopeQuery: String) {
-			image(id: $imageID) {
+		query getImageComponents($query: String, $scopeQuery: String) {
+            imageComponents(query: $query, scopeQuery: $query) {
```
Comment (md2119) [inline tests/active_vuln_test.go:188]: `scopeQuery` var seems to be ignored.

File: tests/active_vuln_test.go [hunk 2]
```diff
@@ -114,119 +108,98 @@ func TestActiveVulnerability_SetImage(t *testing.T) {
...
+func getImageVulnerabilities(t *testing.T, query string, scopeQuery string) []ActiveVulnerability {
 	var resp struct {
-		Deployment ComponentsAndVulnsWithActiveState `json:"deployment"`
+		Vulnerabilities []ActiveVulnerability `json:"imageVulnerabilities"`
 	}
 	makeGraphQLRequest(t, `
-		query getDeploymentCVE($deploymentID: ID!) {
-			deployment(id: $deploymentID) {
+		query getImageVulnerabilities($query: String, $scopeQuery: String) {
+            imageVulnerabilities(query: $query, scopeQuery: $query) {
```
Comment (md2119) [inline tests/active_vuln_test.go:165]: scopeQuery var seems to be ignored.

File: tests/active_vuln_test.go (hunk 1 above)
Comment (c-du) [reply tests/active_vuln_test.go:188]: scopeQuery is for activeState

**Review by dashrews78 — APPROVED** [review]

**Pull request merged**